src/KubeOps/Operator/Services/ResourceTypeService.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using System.Threading.Tasks;
+
+namespace KubeOps.Operator.Services
+{
+    public class ResourceTypeService : IResourceTypeService
```

Comment:
```suggestion
    internal class ResourceTypeService : IResourceTypeService
```
to open a class later on to public is a feature, to close it down would be a breaking change. because of this, I try to use internal as much as possible for implementations and open up needed classes when the usecase arises.

Reply:
Fixed